backlogger.py

```diff
@@ -18,6 +18,7 @@
 # Icons used for PASS or FAIL in the md file
 result_icons = {"pass": "&#x1F49A;", "fail": "&#x1F534;"}
 reminder_text = "This ticket was set to **{priority}** priority but was not updated [within the SLO period]({url}). Please consider picking up this ticket or just set the ticket to the next lower priority."
+update_slo_text = "The ticket will be set to the next lower priority **{priority}**. [Further info]({url})"
```

Comment:
I suggest to simply stick to the already defined templates from https://progress.opensuse.org/projects/openqatests/wiki#SLOs-service-level-objectives . So the first sentence stays always the same. Only the second sentence differs for first/second reminder. This way it's obvious that both reminders belong together and are part of the same process flow.

Reply:
Same for me, your suggestion from https://github.com/openSUSE/backlogger/pull/52#discussion_r1599781909 reads good to me